esp/services/ldapenvironment/ldapenvironmentService.cpp

```diff
@@ -362,6 +363,22 @@ bool CldapenvironmentEx::onLDAPCreateEnvironment(IEspContext &context, IEspLDAPC
                 notes.appendf("\nNon Fatal Error creating '%s'", respLDAPAdminUser.str());
             }
 
+            //Add LDAPAdmin user to Administrators group
+            if (secmgr->getLdapServerType() == ACTIVE_DIRECTORY)
+            {
+                const char * pDC = strstr(respUsersBaseDN.str(), "dc=");
+                VStringBuffer ldapadminGrpOU("cn=Administrators,cn=Builtin,%s", pDC ? pDC : "dc=local");
+                VStringBuffer ldapadminUsr("%s%s,%s", userPrefix, respLDAPAdminUser.str(), respUsersBaseDN.str());
```

Comment:
note: This is the same as ldapAdminFQDM in line 384, and would be clearer if it was shared  Will merge as-is.

Reply:
Good catch @Gavin, I'll fix this in a follow up PR